abe/helper_functions/ics_helpers.py

```diff
@@ -3,30 +3,23 @@
 helpful inspiration: https://gist.github.com/jason-w/4969476
 """
 
+import datetime
 import logging
-import pdb
-import pytz
-
-from icalendar import Calendar, Event, vCalAddress, vText, vDatetime, Timezone
-from dateutil.rrule import rrule, MONTHLY, WEEKLY, DAILY, YEARLY
-import datetime  # import datetime, timedelta, timezone, date, time
-from datetime import timedelta, timezone, date
-from bson import objectid
-from mongoengine import *
-from icalendar import Calendar
+from datetime import time, timedelta, timezone
 
 import dateutil.parser
+import pytz
 import requests
 from icalendar import Calendar
-from icalendar import Event
+import icalendar as ical
 
 from abe import database as db
 from abe.helper_functions.converting_helpers import mongo_to_dict
-from abe.helper_functions.sub_event_helpers import create_sub_event, update_sub_event, sub_event_to_full, \
-    find_recurrence_end
+from abe.helper_functions.sub_event_helpers import (create_sub_event, find_recurrence_end, sub_event_to_full,
+                                                    update_sub_event)
 
 
-def create_ics_event(event: db.Event, recurrence=False) -> Event:
+def create_ics_event(event: db.Event, recurrence=False) -> ical.Event:
```

Comment:
I found the use of (bare) `Event` confusing in this file, so I made them all qualified (`db.Event` vs. `ical.Event`).